webrtc-stats.html

```diff
@@ -410,7 +410,7 @@
         </h3>
         <div>
           <pre class="idl">dictionary RTCRTPStreamStats : RTCStats {
-             DOMString     ssrc;
+             unsigned long ssrc;
```

Comment:
Please align with the others, I don't think the formatting will re-align your webIDL.